### PR TITLE
Create a shitload of enums

### PR DESCRIPTION
**Pull request by R-unic**

### CODE REVIEW

File: NIF/Core/Types/PixelComponent.cs [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+        /// <summary>
+        /// Stencil component.
+        /// </summary>
+        COMP_STENCIL = 18,
+        /// <summary>
+        /// Empty component.
+        /// </summary>
+        COMP_EMPTY = 19
+    }
+}
```
Comment (glomdom) [inline NIF/Core/Types/PixelComponent.cs:87]: woah. this is nice!

File: NIF/Core/Types/PixelTiling.cs [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+    /// <summary>
+    /// Describes whether pixels have been tiled from their standard row-major format to a format optimized for a particular platform.
+    /// </summary>
+    public enum PixelTiling : uint {
+        TILE_NONE = 0,
+        TILE_XENON = 1,
+        TILE_WII = 2,
+        TILE_NV_SWIZZLED = 3
+    }
+}
```
Comment (glomdom) [inline NIF/Core/Types/PixelTiling.cs:11]: woah. this is nice!

File: NIF/Core/Types/PixelFormat.cs [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+        /// <summary>
+        /// Indicates the NiPixelFormat is meant to be used on a depth/stencil surface.
+        /// </summary>
+        PX_FMT_DEPTH_STENCIL = 15,
+        /// <summary>
+        /// Unknown pixel format.
+        /// </summary>
+        PX_FMT_UNKNOWN = 16
+    }
+}
```
Comment (glomdom) [inline NIF/Core/Types/PixelFormat.cs:75]: woah. this is nice!

**Review by glomdom — APPROVED** [review]
YES.

File: NIF/Core/Types/ApplyMode.cs [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+        /// <summary>
+        /// PS2 Only. Function Unknown.
+        /// </summary>
+        APPLY_HILIGHT,
+        /// <summary>
+        /// Parallax Flag in some Oblivion meshes.
+        /// </summary>
+        APPLY_HILIGHT2
+    }
+}
```
Comment (glomdom) [inline NIF/Core/Types/ApplyMode.cs:27]: woah. this is nice!

File: NIF/Core/Types/AccumFlags.cs [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+        /// <summary>
+        /// +Z is front facing.
+        /// </summary>
+        ACCUM_Z_FRONT,
+        /// <summary>
+        /// -X is front facing.
+        /// </summary>
+        ACCUM_NEG_FRONT,
+    }
+}
```
Comment (glomdom) [inline NIF/Core/Types/AccumFlags.cs:47]: woah. this is nice!

Comment (glomdom) [reply NIF/Core/Types/AccumFlags.cs:47]: damn right it is

File: NIF/Core/Types/NifVersion.cs [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+        V20_3_1_2 = 0x14030102,
+        V20_5_0_0 = 0x14050000,
+        V20_6_0_0 = 0x14060000,
+        V20_6_0_2 = 0x14060002,
+        V30_0_0_2 = 0x1E000002,
+        V30_1_0_1 = 0x1E010001,
+        V30_1_0_3 = 0x1E010003,
+        V30_2_0_3 = 0x1E020003
+    }
+}
```
Comment (glomdom) [inline NIF/Core/Types/NifVersion.cs]: woah. this is NOT NICE WRONG ENUM NAME

File: NIF/Core/Types/FileVersion.cs [hunk 7]
```diff
@@ -1,5 +1,5 @@
 ﻿namespace NIF.Core.Types {
-    public enum NifVersion {
+    public enum FileVersion {
```
Comment (glomdom) [inline NIF/Core/Types/FileVersion.cs]: woah. this is nice!

File: NIF/Core/Types/RendererID.cs [hunk 8]
```diff
@@ -0,0 +1,14 @@
...
+    public enum RendererID {
+        XBOX360,
+        PS3,
+        DX9,
+        D3D10,
+        WII,
+        GENERIC,
+        D3D11
+    }
+}
```
Comment (glomdom) [inline NIF/Core/Types/RendererID.cs:14]: woah. this is nice!

File: NIF/Core/Types/PlatformID.cs [hunk 9]
```diff
@@ -0,0 +1,16 @@
...
+        XENON,
+        PS3,
+        DX9,
+        WII,
+        D3D10,
+        UNKNOWN_6,
+        UNKNOWN_7,
+        UNKNOWN_8
+    }
+}
```
Comment (glomdom) [inline NIF/Core/Types/PlatformID.cs:16]: woah. this is nice!

File: NIF/Core/Types/BipedPart.cs [hunk 10]
```diff
@@ -1,7 +1,4 @@
 ﻿namespace NIF.Core.Types {
-    /// <summary>
-    /// Describes the options for the accum root on NiControllerSequence.
-    /// </summary>
     public enum BipedPart {
         /// <summary>
         /// Other
```
Comment (glomdom) [inline NIF/Core/Types/BipedPart.cs:4]: dont delete code..

File: NIF/Core/Types/BipedPart.cs [hunk 11]
```diff
@@ -0,0 +1,123 @@
...
+        /// <summary>
+        /// AddonLeg
+        /// </summary>
+        P_ADDON_LEG,
+        /// <summary>
+        /// AddonArm
+        /// </summary>
+        P_ADDON_ARM
+    }
+}
```
Comment (glomdom) [inline NIF/Core/Types/BipedPart.cs:120]: woah. this is nice!

Comment (glomdom) [reply NIF/Core/Types/BipedPart.cs:120]: and long

File: NIF/Core/Types/TexType.cs [hunk 12]
```diff
@@ -1,6 +1,6 @@
 ﻿namespace NIF.Core.Types {
     /// <summary>
-    /// Describes how the vertex colors are blended with the filtered texture color.
+    /// The type of texture.
```
Comment (glomdom) [inline NIF/Core/Types/TexType.cs:3]: woah. this is nice!

File: NIF/Core/Types/KeyType.cs [hunk 13]
```diff
@@ -0,0 +1,27 @@
...
+        /// <summary>
+        /// For use only with rotation data. Separate X, Y, and Z keys will be stored instead of using quaternions.
+        /// </summary>
+        XYZ_ROTATION_KEY,
+        /// <summary>
+        /// Step function. Used for visibility keys in NiBoolData.
+        /// </summary>
+        CONST_KEY
+    }
+}
```
Comment (glomdom) [inline NIF/Core/Types/KeyType.cs:27]: woah. this is nice!

File: NIF/Core/Types/TexType.cs [hunk 14]
```diff
@@ -0,0 +1,55 @@
...
+        /// <summary>
+        /// For placing images on the object like stickers.
+        /// </summary>
+        DECAL_2_MAP,
+        /// <summary>
+        /// For placing images on the object like stickers.
+        /// </summary>
+        DECAL_3_MAP
+    }
+}
```
Comment (glomdom) [inline NIF/Core/Types/TexType.cs:55]: THIS right here is the BEST

**Comment by glomdom** [comment]
Dear Mr. R-Unic,

I hope this message finds you in high spirits and good health. Today, I write not just as a colleague or project lead, but as someone who has had the immense privilege of witnessing your journey up close within our epic project. Your first pull request is not just a milestone for the project, but a testament to your unwavering commitment, exceptional skill, and the innovative spirit you bring to our collective table.

From the outset, you've been more than a contributor; you've been the backbone of our endeavor, threading your insights, expertise, and unique perspective into the very fabric of our work. Your dedication has not only propelled the project forward but has also inspired a sense of unity and purpose among us all.

This pull request symbolizes more than just code; it represents the countless hours of learning, the challenges overcome, and the shared moments of triumph and discovery. It's a reflection of your passion, your resilience, and your invaluable contribution to not just the project, but to the very essence of our team spirit.

As we move forward, I want you to know that your efforts are deeply appreciated and recognized. You're not just shaping the project; you're crafting a legacy that will inspire future contributors, encouraging them to pursue excellence with the same fervor and dedication you've shown.

Thank you for being an integral part of this journey. Here's to many more milestones, shared successes, and the continued growth of our project. Your first pull request is just the beginning, and I eagerly anticipate the innovations and accomplishments that lie ahead.

With heartfelt gratitude and respect,

glomdom

**Comment by glomdom** [comment]
DO NOT SWEAR ON THE PULL REUQESTS.

File: NIF/Core/Types/BipedPart.cs (hunk 10 above)
Comment (R-unic) [reply NIF/Core/Types/BipedPart.cs:4]: LMFAO

**Comment by R-unic** [comment]
> DO NOT SWEAR ON THE PULL REUQESTS.

Oops

**Pull request merged**